### DISCUSSION TITLE
opencv_word.so.4.11这种lib库应该怎么添加？编译时找不到

### DISCUSSION THREAD

**Discussion by cyinen** [xmake-io/xmake#1991] [Q&A]
![image](https://user-images.githubusercontent.com/56926538/150062610-2b5729cc-bf94-40ec-ba96-f931449f8e46.png)

我想使用opencv的lib库，但是提示找不到。请问我应该如何正确添加

**Comment by waruqi** [comment]
add_requires("opencv") 走包管理，具体使用看文档
